lewis/examples/example_motor/__init__.py

```diff
@@ -146,3 +146,5 @@ def set_target(self, new_target):
         )
     )
 )
+
+framework_version = '>=1.0.3'
```

Comment:
This should probably be reduced to `1.0.2` for now, like the others, and the ranging removed.